lizmap/modules/presentation/classes/presentation.listener.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+class presentationListener extends jEventListener
+{
+    public function ongetMapAdditions($event)
+    {
+        $basePath = jApp::config()->urlengine['basePath'];
+
+        // Add JS and CSS for module
+        $jsCode = array();
+        $css = array();
+
+        // Check config
+        jClasses::inc('presentation~presentationConfig');
+        $presentationConfigInstance = new presentationConfig($event->repository, $event->project);
+        if ($presentationConfigInstance->getStatus()) {
+            $presentationConfig = $presentationConfigInstance->getConfig();
+            $presentationConfigData = array(
+                'url' => jUrl::get(
+                    'presentation~service:index',
+                    array(
+                        'repository' => $event->repository,
+                        'project' => $event->project,
+                    )
+                ),
+            );
+
+            $jsCode = array(
```

Comment:
please stop embedding JS or CSS directly into the HTML, it is not anymore a good practice, and it avoid us to have good secured CSP.  For dynamic JS, create an action that will forge the JS, and declare the address of the action here.